packages/themes/src/index.ts

```diff
@@ -91,8 +92,8 @@ export const themeLabels: Record<ThemeId, string> = {
   mars: 'Mars',
   deepSpace: 'Deep Space',
   laserwave: 'Laserwave',
-  none: '',
-}
+  none: 'None',
```

Comment:
Seems like we exclude this from the list in most places anyway but it should probably actually have a proper label.